03_summarize/src/plot_functions.R

```diff
@@ -0,0 +1,26 @@
+#' Plot timeseries for a given site and characteristic
+#'
+#' @param site_id A character string describing a site ID for a site in the input
+#' dataframe
+#' @param df A dataframe containing water quality data for one characteristic
+#' for one or more sites from the Water Quality Portal
+#' @param out_path A folder path to the location where plots should be saved.
+#'
+#' @return Saves a .png plot at the `out_path` location and returns the file 
+#' path to the plot.
+
+plot_timeseries <- function(site_id, df, out_path) {
+  site_data <- df |>
+    filter(MonitoringLocationIdentifier %in% site_id)
+  
+  ggplot(site_data, aes(x = date, y = result_value)) +
+    geom_point() +
+    labs(title = paste(unique(site_data$MonitoringLocationIdentifier), " - ", unique(site_data$CharacteristicName)),
+         x = "Date",
+         y = "mg/L") +
+    theme_classic()
+  
+  file_path <- paste0(out_path, "/timeseries_", site_id, ".png")
+  
+  ggsave(file_path)
```

Comment:
I think we should choose the specific size to 1) avoid the console message about saving an image, and 2) highlight where/how someone could change the image size if `ggsave()` is new to them.

```suggestion
  ggsave(file_path, width = 7, height = 7)
```

Reply:
Good idea.